sphere/model/abstract/model.py

```diff
@@ -25,10 +25,6 @@ def __init__(self, params: Parameters, solver: ODESolver):
     def state_transition(self, state: ArrayLike, t: int) -> Array:
         pass
 
-    @abstractmethod
-    def observation(self, state: ArrayLike, t: int) -> Array:
```

Comment:
I think we might want to consider keeping this.

Reply:
i added it back in new commit.